packages/agent/src/display.ts

```diff
@@ -7,6 +7,8 @@ import { ClaimFormat, Hasher, JsonTransformer, SdJwtVcRecord } from '@credo-ts/c
 import { getHostNameFromUrl, sanitizeString } from '@package/utils'
 import { decodeSdJwtSync, getClaimsSync } from '@sd-jwt/decode'
 
+import type { SeedCredentialPidData } from '@ausweis/storage'
```

Comment:
I think the dependencies are not great here. ausweis depends on agent, not the other way around. As we use a separate `usePidCredential` only available in ausweis wallet my assumption was that we will add all this SEED credential stuff only to the ausweis app. Is that OK?